projects/Mallard/src/services/__tests__/logging.spec.ts

```diff
@@ -61,9 +61,29 @@ describe('logging service', () => {
             })
             loggingService.clearLogs = jest.fn()
             loggingService.postLog = jest.fn()
+            loggingService.hasConsent = true
             await loggingService.log({ level: Level.INFO, message: 'test' })
             expect(loggingService.postLog).toHaveBeenCalled()
             expect(loggingService.clearLogs).toHaveBeenCalled()
         })
+        it('should not post a log if there is no consent', async () => {
+            loggingService.getExternalInfo = jest.fn().mockReturnValue({
+                networkStatus: { type: 'wifi' },
+                userData: {
+                    userDetails: { id: 'testId' },
+                    membershipData: {
+                        contentAccess: { digitalPack: true },
+                    },
+                },
+                casCode: 'QWERTYUIOP',
+                iapReceipt: true,
+            })
+            loggingService.clearLogs = jest.fn()
+            loggingService.postLog = jest.fn()
```

Comment:
When I comment out above two lines test fails. It should be irreverent when `hasConsent` is set to `false`. Is this expected behavior? (p.s. I may not understand js test properly)

Reply:
The previous test has set it to true and we are no re-instationating the logging service. Can be moved to a before each?